### PR TITLE
Fix issue with retrieving events in batch edit

### PR DESCRIPTION
**Pull request by bp323**
Currently it doesn't look like all events are being retrieved and displayed in batch edit mode.

The number of events to retrieve should always be reasonable enough to just make looped requests until all data has been retrieved. If paging is required this will have to go through design first.

### CODE REVIEW

**Comment by bp323** [comment]
Reassigning to @Coenego for review

File: shared/gh/js/views/gh.admin-batch-edit.js [hunk 1]
```diff
@@ -761,12 +761,41 @@ define(['gh.constants', 'gh.utils', 'gh.api.event', 'gh.api.groups', 'gh.api.ser
...
+             */
+            var getSeriesEvents = function(offset, callback) {
+                // Get the information about the events in the series
+                seriesAPI.getSeriesEvents(seriesId, 25, offset, false, function(err, _events) {
+                    if (err) {
+                        return gh.utils.notification('Events not retrieved.', 'The events could not be successfully retrieved.', 'error');
+                    }
+
+                    // Aggregate the results
+                    events.results = events.results.concat(_events.results);
```
Comment (Coenego) [inline shared/gh/js/views/gh.admin-batch-edit.js]: We can use the `_.union()` function here

File: shared/gh/js/views/gh.admin-batch-edit.js [hunk 2]
```diff
@@ -761,12 +761,41 @@ define(['gh.constants', 'gh.utils', 'gh.api.event', 'gh.api.groups', 'gh.api.ser
...
+                    } else {
+                        callback();
+                    }
+                });
+            };
 
+            // Get the first page of events
+            getSeriesEvents(0, function() {
                 // Unlock the currently locked group
                 unlockGroup();
```
Comment (Coenego) [inline shared/gh/js/views/gh.admin-batch-edit.js]: Won't the `gh.admin.changeView` event be dispatched every time we loop this function as well then? Preferably this should only be dispatched once after all the events were retrieved from the back-end.

**Comment by Coenego** [comment]
Left a question and a comment. Reassigning to @bp323

File: shared/gh/js/views/gh.admin-batch-edit.js (hunk 2 above)
Comment (bp323) [inline shared/gh/js/views/gh.admin-batch-edit.js]: The callback is only executed after all events have been retrieved, unless I'm missing something this shouldn't be triggered more than once.

**Comment by bp323** [comment]
Back to @Coenego for review

**Pull request merged**